apps/ledger-live-desktop/src/renderer/families/polkadot/ManageModal.tsx

```diff
@@ -126,10 +127,11 @@ const ManageModal = ({ account, source, ...rest }: Props) => {
   );
 
   const electionOpen = staking?.electionClosed !== undefined ? !staking?.electionClosed : false;
+  const accountCanNominate = canNominate(account);
   const hasUnlockedBalance = unlockedBalance && unlockedBalance.gt(0);
   const hasPendingWithdrawUnbondedOperation = hasPendingOperationType(account, "WITHDRAW_UNBONDED");
-  const nominationEnabled = !electionOpen && canNominate(account);
-  const chillEnabled = !electionOpen && canNominate(account) && nominations?.length;
+  const nominationEnabled = !electionOpen && accountCanNominate;
+  const chillEnabled = !electionOpen && isStashAccount(account) && nominations?.length;
```

Comment:
If i'm not mistaken, this is the only real update in terms of behavior, right ? (same for LLM)

I think we should not change isController to isStashAccount, IMO it is misleading for the dev, we should use hasStash name (as it is actually what the method does : check if stash is defined)

History : Controller is an account that can control a stash account, acting as a staking proxy so we don't sign those tx on the stash account directly

This mechanism has been discontinued but this doesn't mean a controller account becomes a stash (if you see what i mean :sweat_smile: )

Reply:
Okay I might I have misunderstood the role of the controller when doing this change then 👍🏿 I'll revert it

Reply:
![image](https://github.com/LedgerHQ/ledger-live/assets/19901996/8e5d6d63-ce63-4c13-89ef-10d938f1f65e)
Are you sure we need to edit the method name ?

It looks like stash is actually the linked stash to the controller
And controller is the controller account linked to this stash

Reply:
Then it means the fix is just removing the minimumBondBalance condition on chillEnabled (which you did, but also edited the method name)

Reply:
So the `isStash` method checks the `controller` property and the `isController` method checks the `stash` property.
I reverted the name to how it was I don't mind but this is confusing to me

Reply:
Yes it's kind of a link between them i guess

If you have a controller it means you are a stash, if you have a stash property it means you are a controller of that stash

Reply:
It's wrote in comments :
```
 * Returns true if account is a stash.
 * When the account is a stash, we have the information of which account is the controller
```